src/node_http2_core-inl.h

```diff
@@ -77,9 +79,11 @@ inline int Nghttp2Session::OnHeaderCallback(nghttp2_session* session,
                                             uint8_t flags,
                                             void* user_data) {
   Nghttp2Session* handle = static_cast<Nghttp2Session*>(user_data);
+  // If this is a push promise frame, we want to grab the handle of
+  // the promised stream.
   int32_t id = (frame->hd.type == NGHTTP2_PUSH_PROMISE) ?
-  frame->push_promise.promised_stream_id :
-  frame->hd.stream_id;
+    frame->push_promise.promised_stream_id :
```

Comment:
These two lines are only indented two spaces vs. the four spaces for the ternary changes up above.